DSharpPlus/GlobalSuppressions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~F:DSharpPlus.Net.RateLimitBucket.UNLIMITED_HASH")]
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~F:DSharpPlus.Net.RateLimitBucket.IsUnlimited")]
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~P:DSharpPlus.Net.RateLimitBucket._resetAfterOffset")]
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~P:DSharpPlus.Entities.DiscordInteractionDataOption._value")]
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~F:DSharpPlus.DiscordClient.DiscordEpoch")]
```

Comment:
Is there any adequate reason to have these?

Reply:
No I'm removing it afterwards

Reply:
OK

Reply:
@Lulalaby want me to remove them now?

Reply:
Done